### DISCUSSION TITLE
How can I hide password?

### DISCUSSION THREAD

**Discussion by hqm199** [centreon-archive/centreon-archived#9808] [Ideas]
In the service details page, we can see the command statement. Sometimes some login user name and password are configured in the command. How can I hide it?
Here is an example of the command:
perl /usr/lib64/nagios/plugins/centreon-plugins/centreon_plugins.pl --plugin=os::windows::snmp::plugin --hostname 192.168.1.82 --snmp-community password--mode uptime --warning-uptime 80 --critical-uptime 90
I want hide password

**Comment by ponchoh** [comment]
Hi @hqm199 

Under the Configuration > 'Custom macros' section you can setup the field as a password

![image](https://user-images.githubusercontent.com/85764737/154564997-3633783b-5011-4777-abfa-a3a41ca366e2.png)

![image](https://user-images.githubusercontent.com/85764737/154565040-8eb6e9ed-dc5b-4bee-ba0c-eab4594f7c7e.png)
